Tools/Scripts/webkitpy/w3c/test_importer.py

```diff
@@ -602,6 +602,10 @@ def import_tests(self):
             self.write_import_log(new_path, copied_files, prefixed_properties, prefixed_property_values)
 
         _log.info('Import complete')
+        _log.info('------- COPY & PASTE in your PR. -------')
```

Comment:
We shouldn't ask people to copy and paste things into PRs, we should put it into the commit message which automatically gets included as part of the PR via the expected `git-webkit` workflow.

Reply:
@gsnedders 
Yes. I was thinking it would be better, but that doesn't seem possible as it is two completely async and disconnected processes. 

Or maybe there's a better way to do it that I have missed. 

1. open a bug on bugs.webkit.org "Import WPT /css/foobar" -> bugid
2. `git checkout -b bugid-css_foobar`
3. `import-w3c-tests -t css/foobar` (here the moment we can get the WPT commit hash)
4. fix what needs to be.
5. `git webkit commit --issue bugid-css_foobar .`← At this stage the WPT commit hash in the commit message that we edit manually
6. `git webkit pr`

Is there a way to store on the filesystem some information for this in the current process. That would be great if possible.

Reply:
Ah, true. I was thinking of something unrelated when I wrote that comment.

Instead of "COPY & PASTE in your PR." can we go with something like "Please include the following in your commit message:"?

Reply:
The message is now 

```
--------- Please include the following in your commit message: ---------

Upstream commit: https://github.com/web-platform-tests/wpt/commit/3d084c0ac239eff15e62958a18b5508203dc2aaa

------------------------------------------------------------------------
```

Reply:
done.